File: update_cards.js
const fs = require('fs');
const path = require('path');

const files = fs.readdirSync('.').filter(f => f.endsWith('.html'));

files.forEach(f => {
    let content = fs.readFileSync(f, 'utf-8');
    
    // Remove active class
    content = content.replace(/class="size-col\s+active"/g, 'class="size-col"');
    
    // Remove onclick="selectCardSize(...)"
    content = content.replace(/\s*onclick="selectCardSize[^"]+"/g, '');
    
    // Replace button
    content = content.replace(
        /<button class="btn-pedir-online[^>]*>PEDIR ONLINE<\/button>/g,
        '<div style="text-align: center; margin-top: 15px; margin-bottom: 5px;"><a href="#productos" class="btn btn-primary"><i class="fa-solid fa-pizza-slice"></i> Pedir online</a></div>'
    );
    
    fs.writeFileSync(f, content, 'utf-8');
});
console.log("Updated HTML files.");

File: update_cards.py
import re
import glob

files = glob.glob('*.html')

for f in files:
    with open(f, 'r', encoding='utf-8') as file:
        content = file.read()
    
    # Remove active class from size-col
    content = re.sub(r'class="size-col\s+active"', 'class="size-col"', content)
    
    # Remove onclick="selectCardSize(...)"
    content = re.sub(r'\s*onclick="selectCardSize[^"]+"', '', content)
    
    # Replace button
    # Original: <button class="btn-pedir-online btn-buy" onclick="addToCart('...', '...')">PEDIR ONLINE</button>
    # Also need to match variations just in case
    content = re.sub(
        r'<button class="btn-pedir-online[^>]*>PEDIR ONLINE</button>',
        r'<div style="text-align: center; margin-top: 15px; margin-bottom: 5px;"><a href="#productos" class="btn btn-primary"><i class="fa-solid fa-pizza-slice"></i> Pedir online</a></div>',
        content
    )

    with open(f, 'w', encoding='utf-8') as file:
        file.write(content)
print("Updated HTML files.")
